rllib/utils/typing.py

```diff
@@ -143,5 +144,21 @@
 # (possibly nested) dict|tuple of gym.space.Spaces.
 SpaceStruct = Union[gym.spaces.Space, dict, tuple]
 
+# A batch of RNN states with dimensions [state_index, batch, state_object].
+StateBatch = List[List[Any]]
+
+# Data type that is feeded into and yielded from agent connectors.
+AgentConnectorDataType = namedtuple(
```

Comment:
would be nice if we could further specify what "data" and "output" below should be.

Reply:
done.